release.json

```diff
@@ -2,7 +2,7 @@
     "base_branch": "main",
     "current_milestone": "7.68.0",
     "dependencies": {
-        "OMNIBUS_RUBY_VERSION": "34bf3ab7bc6ab0f1a08dee724590dcf0f76a443a",
+        "OMNIBUS_RUBY_VERSION": "49bffd5cbf5f4fc8a357afca145482111c8880ee",
```

Comment:
I am curious about this change. The other dependencies seems to include both version and hash. This just uses an opaque value.  I'm guessing this is a git commit hash. Just using a hash does not give an easy human verifiable path back to release versions. And, if we are not pointing to explicit releases, then we have a different problem.

Reply:
Both this `OMNIBUS_RUBY_VERSION` as well as `INTEGRATIONS_CORE_VERSION` point at commits, from [DataDog/omnibus-ruby](https://github.com/DataDog/omnibus-ruby) and [DataDog/integrations-core](https://github.com/DataDog/integrations-core) respectively.

These repositories don't have "explicit releases", but they do get tags on Agent release branches and this file points to those on release branches ([example](https://github.com/DataDog/datadog-agent/blob/a0b58897f677f754700c84eca9cdbf1706f483af/release.json#L5-L6)). But it's currently either/or, this `VERSION` thing is essentially a git ref.